lib/membrane_flv_plugin/muxer.ex

```diff
@@ -133,23 +133,25 @@ defmodule Membrane.FLV.Muxer do
       dts: timestamp
     }
     |> prepare_to_send(state)
-    |> then(fn {actions, state} -> {{:ok, actions}, state} end)
   end
 
   @impl true
-  def handle_caps(Pad.ref(type, _id) = _pad, caps, _ctx, _state),
-    do: raise("Caps `#{inspect(caps)}` are not supported for stream type #{inspect(type)}")
+  def handle_stream_format(Pad.ref(type, _id) = _pad, stream_format, _ctx, _state),
+    do:
+      raise(
+        "stream_format `#{inspect(stream_format)}` are not supported for stream type #{inspect(type)}"
```

Comment:
Sounds a little weird
```suggestion
        "Format `#{inspect(stream_format)}` is not supported for stream type #{inspect(type)}"
```

Reply:
Not my intention it was, find & replace 😛